Uplift/Views/SportsDetail/SportsDetailInfoCollectionViewCell.swift

```diff
@@ -0,0 +1,106 @@
+        super.init(frame: frame)
+        dateLabel.font = ._16MontserratLight
+        dateLabel.textAlignment = .center
+        dateLabel.numberOfLines = 0
+        dateLabel.textColor = .primaryBlack
+        contentView.addSubview(dateLabel)
+        
+        timeLabel.font = ._16MontserratMedium
+        timeLabel.textAlignment = .center
+        timeLabel.numberOfLines = 0
+        timeLabel.textColor = .primaryBlack
+        contentView.addSubview(timeLabel)
+        
+        joinButton.setTitle(GameStatus.open.rawValue, for: .normal)
+        joinButton.setTitleColor(.primaryBlack, for: .normal)
+        joinButton.titleLabel?.font = ._14MontserratBold
+        joinButton.layer.borderColor = UIColor.primaryBlack.cgColor
+        joinButton.layer.borderWidth = 1
+        joinButton.layer.cornerRadius = 20
+        joinButton.layer.masksToBounds = true
+        joinButton.addTarget(self, action: #selector(joinButtonPressed), for: .touchUpInside)
+        contentView.addSubview(joinButton)
+        
+        divider.backgroundColor = .gray01
+        contentView.addSubview(divider)
+        
+        setupConstraints()
+    }
+    
+    @objc func joinButtonPressed() {
+        // TODO: toggle join button.
+    }
+    
+    func setupConstraints() {
+        let horizontalPadding = 40
+        let dateLabelTopPadding = 24
+        let dateTimeVerticalPadding = 8
+        let joinButtonBottomPadding = 24
+        
+        dateLabel.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(horizontalPadding)
+            make.top.equalToSuperview().offset(dateLabelTopPadding)
+        }
+        
+        timeLabel.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(horizontalPadding)
+            make.top.equalTo(dateLabel.snp.bottom).offset(dateTimeVerticalPadding)
+        }
+        
+        joinButton.snp.makeConstraints { make in
+            make.width.equalTo(132)
+            make.height.equalTo(41)
+            make.bottom.equalToSuperview().inset(joinButtonBottomPadding)
+            make.centerX.equalToSuperview()
+        }
+        
+        divider.snp.makeConstraints { make in
+            make.bottom.leading.trailing.equalToSuperview()
+            make.height.equalTo(1)
+        }
+    }
+    
+    func configure(for post: Post) {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "MMM dd"
+        // TODO: add game date field to post, and replace created at with game date.
+        var weekday = ""
+        switch Calendar.current.component(.weekday, from: post.createdAt) {
+            case 1: weekday = "Sunday"
+            case 2: weekday = "Monday"
+            case 3: weekday = "Tuesday"
+            case 4: weekday = "Wednesday"
+            case 5: weekday = "Thursday"
+            case 6: weekday = "Friday"
+            case 7: weekday = "Saturday"
+            default: weekday = ""
+        }
```

Comment:
maybe also move this into `Date+Helpers.swift` since this seems like something we use a lot